arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -1068,17 +1084,9 @@
           "readOnly": true
         },
         "provisioningState": {
```

Comment:
We have noticed in several places that `provisioningState` is not modeled as enums such as in [ContainerService](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-compute/2017-01-31/swagger/containerService.json#L544), [ARM resource groups](https://github.com/AppPlatPerf/azure-rest-api-specs/blob/master/arm-resources/resources/2017-05-10/swagger/resources.json#L2389), etc. At this point we expect to have more changes on what values we will have so we hope to keep it this way rather than putting down a few values and then making swagger changes in the near future. We have also asked ARM team but is this an acceptable approach?